UPGRADE-6.3.md

```diff
@@ -298,14 +298,16 @@ is recommended (although this tool may not find database-related deprecations).
 - \ChameleonSystem\CoreBundle\Service\TransformOutgoingMailTargetsService::setSubjectPrefix()
 - \MTTableManager::getAutocompleteRecordList()
 - \TCMSCronJob::getLogger()
+- \TCMSFieldColorPicker::isFirstInstance()
 - \TCMSFieldLookup::enableComboBox()
 - \TCMSLogChange::getUpdateLogger()
+- \TCMSRender::DrawBoxHeader()
```

Comment:
I'd like to deprecate the main menu in the sidebar PR, so that this is "encapsulated". Would you mind to revert related deprecations?

Reply:
o.k. done.